app/controllers/form_controller.rb

```diff
@@ -101,6 +101,14 @@ def responses_for_page(page)
       else
         result[question.id] = question_params
       end
+
```

Comment:
I don't know where this is in the path, but is there a risk of a support user creating, or trying to create a log for an org that hasn't accepted the data agreement?